book/controller.rst

```diff
@@ -5,7 +5,7 @@ Controller
 ==========
 
 A controller is a PHP function you create that takes information from the
-HTTP request and constructs and returns an HTTP response (as a Symfony
+HTTP request to construct and return an HTTP response (as a Symfony
```

Comment:
Well, it's more about grammar than word usage. I think *A controller* is still the one doing stuff:

A controller:
* takes information from the HTTP request
* constructs an HTTP response
* returns the HTTP response

So imo, it's still "takes information from the HTTP request and creates and returns an HTTP response".